java/maven.indexer/external/lucene-9.10.0-license.txt

```diff
@@ -1,11 +1,11 @@
 Name: Apache Lucene
 Description: Java-based indexing and search technology
-Version: 9.9.1
+Version: 9.10.0
 Origin: Apache Software Foundation
 License: Apache-2.0
 URL: https://lucene.apache.org/
 Source: https://github.com/apache/lucene
-Files: lucene-analysis-common-9.9.1.jar lucene-backward-codecs-9.9.1.jar lucene-core-9.9.1.jar lucene-highlighter-9.9.1.jar lucene-queryparser-9.9.1.jar
+Files: lucene-analysis-common-9.10.0.jar lucene-backward-codecs-9.10.0.jar lucene-core-9.10.0.jar lucene-highlighter-9.10.0.jar lucene-queryparser-9.10.0.jar
 
                                  Apache License
```

Comment:
So, while here, it would probably be good to fix the license text. If I look e.g. inside the `lucene-analysis-common-9.10.0.jar`, I see `META-INF/LICENSE.txt`, which contains a lot more text than what's here (as here is Apache 2.0 only).

As a side note, the license text in `lucene-analysis-common-9.10.0.jar/META-INF/LICENSE.txt` is not completely correct, to my knowledge: the NOTICE.txt says this:
> The German,Spanish,Finnish,French,Hungarian,Italian,Portuguese,Russian and Swedish light stemmers
(common) are based on BSD-licensed reference implementations created by Jacques Savoy and
Ljiljana Dolamic. These files reside in:

at least some of the files listed below this note seem to be included in the jar (in compiled form), but there's no license text for this in LICENSE.txt. Based on history, I assume we want be fixing those, but its worth noting, I think.

Reply:
I can try to improve this. So far I copied the license of `lucene-analysis-common` into our license file which includes more sections and also the BSD section. I am not quite sure what else i should do.

Reply:
diff https://github.com/apache/netbeans/compare/68fffd85264348a8e5b5ce594d73912c870cdf14..1e68855a358ad3d68940ddab75a238701de53ad7

Reply:
I think having the license match what's in the jars in acceptable for now. We need to file a bug against Lucene to fix their license at some point.